codecov.yaml

```diff
@@ -1,13 +1,15 @@
 coverage:
+  precision: 2
+  round: up
+  range: "45...70"
   status:
     project:
       default:
         target: auto
-        # allow coverage to drop by this amount and still post success
-        threshold: 0.5%
-        base: auto
-        branches:
-          - master
-        if_not_found: success
-        if_ci_failed: error
-        only_pulls: false
+        threshold: 5%
+        informational: true
```

Comment:
defining `threshold` **and** `informational` does not make sense to me.
Either we want it to succeed *always* or only if coverage does not drop by a lot.
Given that 5% coverage drop in the whole MoveIt repository with a single PR should be pretty hard to do, I would propose to comment out `informational` for now.

Reply:
OK, I'll remove the informational.